pkg/database/authorized_app.go

```diff
@@ -61,8 +70,22 @@ type AuthorizedApp struct {
 	// APIKey is the HMACed API key.
 	APIKey string `gorm:"type:varchar(512);unique_index"`
 
-	// APIKeyType s the API key type.
+	// APIKeyType is the API key type.
 	APIKeyType APIUserType `gorm:"default:0"`
+
+	// OS is the type of the application we're using (eg, iOS, Android).
+	OS OsType `gorm:"os,type:int"`
```

Comment:
Having these on API keys isn't going to work - because API keys will be rotated eventually.
the current positioning would actually let a realm have multiple apps per-platform claimed, and I don't think we want that either.

This information should only be set once on the realm.

Reply:
Moved to a non-per-realm basis.